contents/tutorials/event-tracking-guide.md

```diff
@@ -2,184 +2,177 @@
-author: ['yakko-majuri']
-date: 2021-08-02
+author: ['ian-vanagas', 'yakko-majuri']
+date: 2022-09-21
 featuredImage: ../images/tutorials/banners/actions.png
 topics: ['configuration']
 ---
 
-_Estimated reading time: 7 minutes_ ☕☕
+- **Level:** Medium 🦔🦔
+- **Estimated reading time:** 12 minutes ☕️☕️
 
-One of the features we're very proud of at PostHog is event autocapture.
+Event tracking is the first step in making a product better (after building the product). Event tracking gathers the data you need to understand the usage of your product, do analysis, and make improvement decisions. Ideally, every time a user takes action, an event can be captured that helps deepen your understanding of the usage of your product.
 
-Autocapture is great because it ensures you don't miss out on any data that could potentially be relevant to your product. If you restructure your frontend, for example, autocapture will continue to collect relevant events without you needing to update your codebase each time. 
+At PostHog, we offer two ways to track events: autocapture and custom events. This tutorial addresses both step-by-step, walking through setting up autocapture, then diving deeper into custom events, and finally refining those custom events to capture the right data you need.
 
-However, for more advanced product analytics, autocapture is not enough. 
+## Setting up autocapture
 
-First, event autocapture can be a bit overwhelming. On high volume instances, where events come in by the thousands every minute, the events table provided by PostHog can contain so much information that it can be difficult to determine exactly what you care about. 
+Autocapture makes it easy to start getting data such as page views, clicks, and submissions from your site through a Javascript snippet. It doesn’t require setting up individual events and automatically captures data from new or changed components. This is useful, for example, if you restructure your frontend, autocapture continues to collect relevant events without you needing to update your codebase.
 
-Second, autocapture is a general solution that provides value for most users, but, to get the most out of PostHog, you should be using custom events. Custom events let you track _exactly_ the things that you care about, ranging from a user hovering over a certain part of your app, to a specific function being called on your backend.
+To set up autocapture:
 
-This tutorial will address these two points, walking you through how to create actions and send relevant custom events step-by-step, so you can make the most out of PostHog.
+1. Go to “Project settings”
+2. Under “Website event autocapture,” copy the code block snippet containing the script tags and Javascript function.
+3. Paste the block into your site’s HTML in the `<head>` tags (ideally at the end). This should be an HTML page that acts as a base or template page (with other scripts your page loads) to ensure all possible events are captured.
```

Comment:
```suggestion
1. Go to “Project settings”

2. Under “Website event autocapture,” copy the code block snippet containing the script tags and Javascript function.

3. Paste the block into your site’s HTML in the `<head>` tags (ideally at the end). This should be an HTML page that acts as a base or template page (with other scripts your page loads) to ensure all possible events are captured.
```
Minor style point (which I ought to document somewhere) but when you've got long list items like this, it's a good idea to add an extra line break between them. It helps legibility so text doesn't feel so bunched up. This is necessary if you've got a list of short items, though.